src/library/trackcollectionmanager.cpp

```diff
@@ -84,16 +84,19 @@ TrackCollectionManager::TrackCollectionManager(
         // Handle signals
         connect(m_pScanner.get(),
                 &LibraryScanner::trackAdded,
-                this,
-                &TrackCollectionManager::slotScanTrackAdded);
+                [this](const TrackPointer& pTrack) {
+                    afterTrackAdded(pTrack);
+                });
```

Comment:
Why not make these slots and connect them directly?

Reply:
Because slots are usually and by convention non-const, but the member functions are const and invoked internally from other const functions.

Reply:
OK.